### PR TITLE
Add some more scholars

### PR DESCRIPTION
**Pull request by jcnorby**

### CODE REVIEW

File: _people/EdwardTunstel.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+first_name: Edward
+last_name: Tunstel
+title: Associate Director and Group Leader
+school: Raytheon Technologies Research Center
+website: https://www.thehistorymakers.org/biography/edward-tunstel
+image: http://techlav.ncat.edu/images/Advisory%20Board/Ed%20Tunstel.jpg
+---
+Dr. Tunstel is an associate director of the Autonomous and Intelligent Systems Department at Raytheon, prior to which he was an Adjunct Professor for the Institute for Intelligent Systems Research and Innovation at Deakin University. His expertise is in robot navigation, fuzzy logic and autonomous behavior-based control, and applications of soft computing techniques to intelligent systems.
```
Comment (aaronjoh) [inline _people/EdwardTunstel.md:9]: Definitely mention JPL as his "prior to that" position

File: _people/StephenGuy.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+first_name: Stephen J.
+last_name: Guy
+title: Associate Professor of Computer Science and Engineering
+school: University of Minnesota
+website: https://www-users.cs.umn.edu/~sjguy/
+image: https://www-users.cs.umn.edu/~sjguy/img/Stephen2012.jpg
+---
+Prof. Guy directs the [Applied Motion Lab](http://motion.cs.umn.edu). His research focuses on the development of artificial intelligence for use in autonomous robotics (e.g., collision avoidance and path planning under uncertainty) and computer simulations of human movement and behavior (e.g., crowd simulation and virtual characters).
+- Guy, Stephen J., et al. "[Clearpath: highly parallel collision avoidance for multi-agent simulation.](http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.148.7460&rep=rep1&type=pdf)" Proceedings of the 2009 ACM SIGGRAPH/Eurographics Symposium on Computer Animation. 2009.
```
Comment (aaronjoh) [inline _people/StephenGuy.md:10]: Is there a better link than citeseer?

File: _people/StephenGuy.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+- Karamouzas, Ioannis, Brian Skinner, and Stephen J. Guy. "[Universal power law governing pedestrian interactions.](https://arxiv.org/pdf/1412.1082.pdf)" Physical review letters. 2014.
```
Comment (aaronjoh) [inline _people/StephenGuy.md:11]: Link to physical review letters not arxiv

File: _people/TahiraReidSmith.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+first_name: Tahira
+last_name: Reid Smith
+title: Associate Professor of Mechanical Engineering
+school: Purdue University
+website: https://engineering.purdue.edu/ME/People/ptProfile?resource_id=74127
+image: https://engineering.purdue.edu/ResourceDB/ResourceFiles/image74131
+---
+Prof. Reid Smith leads the [Research in Engineering and Interdisciplinary Design (REID) Lab](https://engineering.purdue.edu/reidlab/) at Purdue University. She studies how to integrate human-centered design into human-machine systems.
+- Akash, Kumar, et al. "[Human Trust-based Feedback Control: Dynamically varying automation transparency to optimize human-machine interactions.](https://arxiv.org/pdf/2006.16353.pdf)" arXiv preprint arXiv:2006.16353 (2020).
+- Hu, Wan-Lin, et al. "[Real-time sensing of trust in human-machine interactions.](https://www.researchgate.net/profile/Kumar_Akash/publication/312545797_Real-Time_Sensing_of_Trust_in_Human-Machine_Interactions/links/59ee290daca272029ddf65a9/Real-Time-Sensing-of-Trust-in-Human-Machine-Interactions.pdf)" IFAC-PapersOnLine 49.32 (2016): 48-53
```
Comment (aaronjoh) [inline _people/TahiraReidSmith.md:11]: Is there a better link than researchgate.net

Comment (jcnorby) [reply _people/TahiraReidSmith.md]: Their lab site linked to a pdf hosted directly by Purdue, which I included. There was also a version on Science Direct if that'd be preferable.

File: _people/EdwardTunstel.md (hunk 1 above)
Comment (jcnorby) [reply _people/EdwardTunstel.md]: See revisions.

File: _people/StephenGuy.md (hunk 2 above)
Comment (jcnorby) [reply _people/StephenGuy.md]: See revisions.

File: _people/StephenGuy.md (hunk 3 above)
Comment (jcnorby) [reply _people/StephenGuy.md]: See revisions.

**Pull request merged**